### PR TITLE
Describe Tileset Hierarchy Choices 

### PR DESCRIPTION
**Pull request by Kilatsat**
This PR is designed to better-explain where EB users can find specific types of polygons within the dataset.

### CODE REVIEW

File: _posts/services/0001-11-01-enterprise-boundaries-v2.md [hunk 1]
```diff
@@ -56,7 +56,11 @@ The Enterprise Boundaries vector tiles do not directly contain detailed feature
 
 ## Tilesets
 
-Enterprise Boundaries are composed of a number of separate tilesets each with their own tileset ID. There are two types of tilesets: lines & areas.
+Enterprise Boundaries are composed of a number of separate tilesets each with their own tileset ID. There are two types of tilesets: lines & areas. For area tilesets, the hierarchy is designed such that layers with lowest numbers generally contain the polygons with the largest areas and levels with the highest numbers contain the polygons with the smallest areas.
```
Comment (colleenmcginnis) [inline _posts/services/0001-11-01-enterprise-boundaries-v2.md]: We actually call them `map ID`s rather than `tileset ID`s in most places... 

image
(^For example the Mapbox Streets tileset in Studio.)

Comment (colleenmcginnis) [reply _posts/services/0001-11-01-enterprise-boundaries-v2.md]: `lines & areas` --> `tilesets containing line features and tilesets containing polygon features`?

Aren't there also point tilesets for centroids?

File: _posts/services/0001-11-01-enterprise-boundaries-v2.md [hunk 2]
```diff
@@ -56,7 +56,11 @@ The Enterprise Boundaries vector tiles do not directly contain detailed feature
 
 ## Tilesets
 
-Enterprise Boundaries are composed of a number of separate tilesets each with their own tileset ID. There are two types of tilesets: lines & areas. For area tilesets, the hierarchy is generally-designed such that layers with lower numbers contain the polygons with the largest areas and levels with the highest numbers contain the polygons with the smallest areas. Administrative levels are anchored to the a0 layer which contains the top-level polygon of each area which contains an ISO 3166-1 code. Postal levels are anchored on the p4 level which is defined as "Full-detail" or "The minimum required number of digits to send mail". 
+Enterprise Boundaries are composed of a number of separate tilesets each with their own tileset ID. There are two types of tilesets: lines & areas. For area tilesets, the hierarchy is designed such that layers with lowest numbers generally contain the polygons with the largest areas and levels with the highest numbers contain the polygons with the smallest areas. 
+<ul>
```
Comment (colleenmcginnis) [inline _posts/services/0001-11-01-enterprise-boundaries-v2.md]: To intro these bullets, maybe:

>Some other notes about tileset hierarchy:

File: _posts/services/0001-11-01-enterprise-boundaries-v2.md (hunk 1 above)
Comment (colleenmcginnis) [reply _posts/services/0001-11-01-enterprise-boundaries-v2.md]: >For area tilesets, the hierarchy is designed such that layers with lowest numbers generally contain the polygons with the largest areas and levels with the highest numbers contain the polygons with the smallest areas. 

I don't think we're using the word `layers` correctly here. Maybe rephrase to something like this:

>[Map IDs](/help/define-map-id/) for Enterprise Boundaries tilesets are in the form `mapbox.enterprise-boundaries-{a|p|s}{level}-{version}`, where: 
>- `{a|p|s}` represents admin, postal, or statistical boundaries.
>- generally, the `{level}` with lowest numbers generally contain the polygons with the largest areas and levels with the highest numbers contain the polygons with the smallest areas.
>- `{version}` is the version of Enterprise Boundaries. All tilesets in this documentation are part of `v2`.

**Comment by colleenmcginnis** [comment]
👋 @Kilatsat is this still relevant? If it is, I'm happy to help translate these changes over to the new layout (see #75). If not, can you close this PR? Thanks!

**Comment by Kilatsat** [comment]
I thought this was resolved back in August... my point is still accurate, but if no one has noticed thus far, never mind I guess.

**Comment by colleenmcginnis** [comment]
Haha @Kilatsat. Not sure how it fell through. I'll incorporate these updates.
